tests/tdscf-1/output.ref

```diff
@@ -0,0 +1,352 @@
+      24A      2.743850  
+
+    Beta Occupied:                                                        
+
+       1A    -11.228970     2A     -0.783471     3A     -0.535233  
+
+    Beta Virtual:                                                         
+
+       4A      0.105281     5A      0.159004     6A      0.221021  
+       7A      0.285269     8A      0.620941     9A      0.737973  
+      10A      0.822031    11A      0.832333    12A      0.906449  
+      13A      1.062089    14A      1.257736    15A      1.303651  
+      16A      1.383572    17A      1.732543    18A      1.810140  
+      19A      1.988311    20A      2.039560    21A      2.087798  
+      22A      2.350790    23A      2.720421    24A      2.769589  
+
+    Final Occupation by Irrep:
+              A 
+    DOCC [     3 ]
+    SOCC [     2 ]
+
+  @UHF Final Energy:   -38.91737871350426
+
+   => Energetics <=
+
+    Nuclear Repulsion Energy =              6.0682991182693025
+    One-Electron Energy =                 -63.7147379959454128
+    Two-Electron Energy =                  18.7290601641718482
+    Total Energy =                        -38.9173787135042630
+
+  UHF NO Occupations:
+  HONO-2 :    3  A 1.9967234
+  HONO-1 :    4  A 1.0000000
+  HONO-0 :    5  A 1.0000000
+  LUNO+0 :    6  A 0.0032766
+  LUNO+1 :    7  A 0.0027828
+  LUNO+2 :    8  A 0.0000008
+  LUNO+3 :    9  A 0.0000000
+
+
+Computation Completed
+
+
+Properties will be evaluated at   0.000000,   0.000000,   0.000000 [a0]
+
+Properties computed using the SCF density matrix
+
+  Nuclear Dipole Moment: [e a0]
+     X:     0.0000      Y:     0.0000      Z:    -0.0000
+
+  Electronic Dipole Moment: [e a0]
+     X:    -0.0000      Y:     0.0000      Z:    -0.2512
+
+  Dipole Moment: [e a0]
+     X:    -0.0000      Y:     0.0000      Z:    -0.2512     Total:     0.2512
+
+  Dipole Moment: [D]
+     X:    -0.0000      Y:     0.0000      Z:    -0.6386     Total:     0.6386
+
+
+*** tstop() called on jeffschriber.local at Thu May 14 16:48:58 2020
+Module time:
+	user time   =       0.49 seconds =       0.01 minutes
+	system time =       0.03 seconds =       0.00 minutes
+	total time  =          0 seconds =       0.00 minutes
+Total time:
+	user time   =       0.49 seconds =       0.01 minutes
+	system time =       0.03 seconds =       0.00 minutes
+	total time  =          0 seconds =       0.00 minutes
+
+
+         ---------------------------------------------------------
+                         TDSCF excitation energies                
+                 by Andrew M. James and Daniel G. A. Smith        
+         ---------------------------------------------------------
+
+  ==> Requested Excitations <==
+
+      9 states with A symmetry
+
+  ==> Options <==
+
+     etol      :              1e-05
+     rtol      :              0.0001
+     guess_type:              DENOMINATORS
+     restricted:              False
+     triplet   :              None
+     ptype     :              rpa
+
```

Comment:
It would be good to have the printed options resemble the input options as closely as possible (or vice versa).
E.g. `r_tol` vs `rtol` (or `r_tolerance` if this is subject to change)  and in particular `pytpe` vs `tda True/False`. Though having the term `rpa` explicitly printed out is valuable for clarity.